lib/octokit/client/organizations.rb

```diff
@@ -335,6 +335,22 @@ def team_by_name(org, team_slug, options = {})
         get "#{Organization.path(org)}/teams/#{team_slug}", options
       end
 
+      # Check team permissions for a repository
+      #
+      # Requires authenticated organization member.
+      #
+      # @param org [String, Integer] Organization GitHub login or id.
+      # @param team_slug [String, Integer] Team slug or Team ID.
+      # @param owner [String] Owner name for the repository.
+      # @param repo [String] Name of the repo to check permissions against.
+      # @return [Sawyer::Resource] Hash representing team permissions for the repository.
+      # @see https://docs.github.com/en/rest/teams/teams#check-team-permissions-for-a-repository
+      # @example
+      #   @client.team_permission_level("github", "justice-league", "octocat", "hello-world", :accept => 'application/vnd.github.v3.repository+json')
+      def team_permission_level(org, team_slug, owner, repo, options = {})
```

Comment:
I'm not convinced by this name, but I'm not sure what the best option would be. What do you think? Some ideas that come to mind:

* `#team_permissions_for_repo`
* `#check_team_permissions_for_repo`
* `#team_permissions_level_for_repo`
* `#check_team_permissions_for_repo`